modules/roles_profiles/manifests/profiles/logging.pp

```diff
@@ -55,16 +57,19 @@
             # https://bugzilla.mozilla.org/show_bug.cgi?id=1520947
         }
         'Darwin': {
+            include macos_utils::set_hostname
```

Comment:
If we were to set the hostname in puppet, we would definitely want to make it a profile to be called out in roles so it isn't buried randomly in the logging profile.  But either way, I don't think puppet should be touching the hostname/scutils.  It should be done once on deployment or actively managed in MDM.